### ISSUE TITLE
Describe Method is not working. 

### ISSUE THREAD

**Issue by MayankCodr** [processing/p5.js-web-editor#2088]
describe('pink square with red heart in the bottom right corner');
// This code is not working. It is throwing an error.
![p5 js_error](https://user-images.githubusercontent.com/96002289/202854282-b14a96c2-c108-4510-8b39-aabd6dface4c.jpg)

**Comment by kjhollen** [comment]
thanks for this bug report @MayankCodr! to help us get a better understanding of the problem, could you also let us know which version of p5.js you're using in this sketch? 

if you don't know where to find the version number, you can open the index.html file (from the left panel in your screenshot) and take a screenshot of what's there. 

the describe() function is still relatively new, so it may not be available in sketches that are linking to an older version of the p5.js library. i can use describe() in new editor sketches made with p5.js version 1.5.0

if you have an older version in your index.html, you can replace the script src lines with the code below to upgrade:

```
    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.5.0/p5.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.5.0/addons/p5.sound.min.js"></script>
```

**Comment by parteekcoder** [comment]
hey @raclim so I think this is not a bug and should be closed by mentioning this issue in docs of p5web-editor that `describe` is present in new version of p5

please tell me so that I can update it in the docs

**Comment by parteekcoder** [comment]
hey @raclim can you please share the repository for the docs

**Comment by parteekcoder** [comment]
@catarak @raclim I think this issue can be closed by just updating the docs
